plugins/katello/nightly/user_guide/remote_databases/index.md

```diff
@@ -168,7 +168,7 @@ Follow the instructions to [prepare remote mongo](plugins/katello/{{ page.versio
 ### Stop the Foreman server
 Stop the Foreman related services to minimize risk of the data changes during the migration
 ```
-katello-service stop
+foreman-maintain stop
 systemctl start postgresql mongod
```

Comment:
This is actually incorrect now that we use the SCL mongodb.

Reply:
Foreman-maintain can handle the service appropriately depending on where it is installed: `foreman-maintain service stop --only mongod`

Reply:
True, but the service name is `rh-mongodb34-mongod` I believe.

Reply:
yup, just giving options, probably better to specify the actual service name though

Looks like we could just reduce the stop and the starting of two services to one line:
```
foreman-maintain service stop --exclude postgresql,rh-mongodb34-mongod
```

which will keep the two services we need running:
```
[root@sat-6-6-qa-rhel7 backup]# foreman-maintain service stop --exclude postgresql,rh-mongodb34-mongod
Running Stop Services
================================================================================
Check if command is run as root user:                                 [OK]
--------------------------------------------------------------------------------
Stop applicable services: 
Stopping the following service(s):

qdrouterd, qpidd, squid, pulp_celerybeat, pulp_resource_manager, pulp_streamer, pulp_workers, smart_proxy_dynflow_core, tomcat, dynflowd, httpd, puppetserver, foreman-proxy
- All services stopped                                                [OK]      
--------------------------------------------------------------------------------

[root@sat-6-6-qa-rhel7 backup]# foreman-maintain service status -b
Running Status Services
================================================================================
Get status of applicable services: 
Displaying the following service(s):

rh-mongodb34-mongod, postgresql, qdrouterd, qpidd, squid, pulp_celerybeat, pulp_resource_manager, pulp_streamer, pulp_workers, smart_proxy_dynflow_core, tomcat, dynflowd, httpd, puppetserver, foreman-proxy
\ displaying rh-mongodb34-mongod                   [OK]                         
\ displaying postgresql                            [OK]                         
\ displaying qdrouterd                             [FAIL]                       
\ displaying qpidd                                 [FAIL]                       
\ displaying squid                                 [FAIL]                       
\ displaying pulp_celerybeat                       [FAIL]                       
\ displaying pulp_resource_manager                 [FAIL]                       
\ displaying pulp_streamer                         [FAIL]                       
\ displaying pulp_workers                          [FAIL]                       
\ displaying smart_proxy_dynflow_core              [FAIL]                       
\ displaying tomcat                                [FAIL]                       
\ displaying dynflowd                              [FAIL]                       
| displaying httpd                                 [FAIL]                       
| displaying puppetserver                          [FAIL]                       
| displaying foreman-proxy                         [FAIL]                       
| All services displayed                                              [FAIL]    
```

So you can do this instead of:
```
foreman-maintain stop
systemctl start postgresql mongod
```

Reply:
That sounds neat. Thank you for the feedback. :+1: